### PR TITLE
Update hybridsearch.md

### PR DESCRIPTION
**Pull request by arthuo**
Translate the English in the document into Chinese

### CODE REVIEW

File: site/zh-CN/userGuide/search/hybridsearch.md [hunk 1]
```diff
@@ -4,20 +4,20 @@ related_key: filter
...
 
-This topic describes how to conduct a hybrid search.
 
-A hybrid search is essentially a vector search with attribute filtering. By specifying [boolean expressions](boolean.md) that filter the scalar fields or the primary key field, you can limit your search with certain conditions.
+当前主题介绍如何进行混合搜索。
 
-The following example shows how to perform a hybrid search on the basis of a regular [vector search](search.md). Suppose you want to search for certain books based on their vectorized introductions, but you only want those within a specific range of word count. You can then specify the boolean expression to filter the `word_count` field in the search parameters. Milvus will search for similar vectors only among those entities that match the expression.
+混合搜索本质上是带有属性过滤的向量搜索。通过指定过滤标量 field 或者主键 field 的 [布尔表达式](boolean.md)，你可以使用特定条件限制搜索。
 
+下面的例子展示了如何在正则 [向量搜索](search.md) 的基础上进行混合搜索。假设你想基于书籍介绍的特征向量搜索某些书籍，然而你只需要特定 word count 范围内的书籍。你可以指定布尔表达式来过滤过滤搜索参数中的 `word_count` field。Milvus 只会在匹配表达式的 entity 中搜索相似的向量。
```
Comment (LocoRichard) [inline site/zh-CN/userGuide/search/hybridsearch.md:15]: 正则 --> 普通
Or simply delete it.

File: site/zh-CN/userGuide/search/hybridsearch.md [hunk 2]
```diff
@@ -4,20 +4,20 @@ related_key: filter
...
 
-## Load collection
 
-All search and query operations within Milvus are executed in memory. Load the collection to memory before conducting a vector search.
+## 加载 collection
+
+Milvus 中的所有搜索和查询操作都在内存中执行。在进行向量相似性搜索之前将 collection 加载到内存中。
```
Comment (LocoRichard) [inline site/zh-CN/userGuide/search/hybridsearch.md:20]: 查询-->结构化匹配

File: site/zh-CN/userGuide/search/hybridsearch.md [hunk 3]
```diff
@@ -157,102 +157,102 @@ Travel Timestamp(Specify a timestamp in a search to get results based on a data
...
-		<th>Description</th>
+		<th>参数</th>
+		<th>说明</th>
 	</tr>
 	</thead>
 	<tbody>
     <tr>
 		<td><code>data</code></td>
-		<td>Vectors to search with.</td>
+		<td>用于搜索的向量</td>
```
Comment (LocoRichard) [inline site/zh-CN/userGuide/search/hybridsearch.md:167]: `。`

File: site/zh-CN/userGuide/search/hybridsearch.md [hunk 4]
```diff
@@ -157,102 +157,102 @@ Travel Timestamp(Specify a timestamp in a search to get results based on a data
...
 	<tbody>
 	<tr>
 		<td><code>collection_name</code></td>
-		<td>Name of the collection to search in.</td>
+		<td>要搜索的 collection 名称。</td>
 	</tr>
 	<tr>
     <td><code>search_params</code></td>
-    <td>Parameters (as an object) used for search.</td>
+    <td>用于搜索的参数（作为对象）。</td>
```
Comment (LocoRichard) [inline site/zh-CN/userGuide/search/hybridsearch.md:218]: ~~作为~~

**Review by LocoRichard — CHANGES_REQUESTED** [review]

**Comment by LocoRichard** [comment]
Hi @arthuo , Thank you! 
I have made a few feedbacks. The others are good enough to be merged.

**Comment by arthuo** [comment]
> Hi @arthuo , Thank you! I have made a few feedbacks. The others are good enough to be merged.

Good morning.I have made changes based on the feedback. Thank you for your work.

**Pull request merged**